src/webgpu/api/validation/encoding/cmds/render/draw.spec.ts

```diff
@@ -459,7 +478,7 @@ success/error as expected. Such set of buffer parameters should include cases li
     }
 
     const calcSetBufferSize = (
-      boundBufferSizeState: 'zero' | 'exile' | 'enough',
+      boundBufferSizeState: 'zero' | 'oneTooSmall' | 'exact',
```

Comment:
nit. maybe just `'tooSmall'` is good enough? Not sure whether the one part is relevant.

Reply:
I think it's kinda helpful to a test reader to know that we're testing the boundary condition. Also both 'zero' and 'oneTooSmall' are too small.